### PR TITLE
Rename multi int store

### PR DESCRIPTION
**Pull request by ahoffer**
Change multi-int-store --> store in select places.
Check the use of the word "ion"
Rename directories and packages.

NOTE: Had to change the version of ingest API to 0.3.1-SNAPSHOT to get it to compile.

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

File: deploy.bash [hunk 1]
```diff
@@ -9,7 +9,7 @@ SET_DOCKER_W=""
 DOCKER_REG=${DOCKER_REGISTRY:-${SET_DOCKER_REG}}
 
 # The name of the preface for the docker images
-DOCKER_NAME="cnxta/cdr-"
+DOCKER_NAME="cnxta/cdr"
```
Comment (emmberk) [inline deploy.bash]: ✏️ Why was this changed?

Comment (ahoffer) [reply deploy.bash]: It looks like a mistake.

**Comment by ahoffer** [comment]
build now

File: ingest/build.gradle [hunk 2]
```diff
@@ -20,7 +20,11 @@ configurations.all {
 dependencies {
     annotationProcessor "org.projectlombok:lombok:1.18.8"
     compileOnly "org.projectlombok:lombok:1.18.8"
+<<<<<<< HEAD
     implementation "com.connexta.ingest:ingest-api-rest-spring-stubs:0.4.0"
+=======
+    implementation "com.connexta.ingest:ingest-api-rest-spring-stubs:0.3.1-SNAPSHOT"
```
Comment (jhunzik) [inline ingest/build.gradle]: ❗️ 👀

**Comment by ahoffer** [comment]
build now
